test/tests/outputs/postprocessor/output_pps_hidden_shown_check.i

```diff
@@ -0,0 +1,90 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  xmin = 0
+  xmax = 1
+  ymin = 0
+  ymax = 1
+  nx = 10
+  ny = 10
+  elem_type = QUAD4
+  # This test uses ElementalVariableValue postprocessors on specific
+  # elements, if you use DistributedMesh the elements get renumbered.
+  parallel_type = replicated
```

Comment:
Glad you bring this up. One of the next bigger and also very important tasks is to get parallel distributed mesh testing up and running. We STILL don't have that target going. @roystgnr is working on fixing a ton of our tests but we should probably give him a hand. We need to get `DistributedMesh` up to first class status.

@andrsd - might be right here but we can't tell with the current testing because we only test DistributedMesh in serial (which makes it very **not** distributed).